src/Types/MutableInputObjectType.php

```diff
@@ -90,6 +91,7 @@ public function getFields(): array
                     if ($fieldDefinition instanceof Type) {
                         $fieldDefinition = ['type' => $fieldDefinition];
                     }
+                    Assert::string($name);
```

Comment:
Why are we using this lib?  What's wrong with `assert(is_string($name))`?

Reply:
fixed